exercises/pangram/canonical-data.json

```diff
@@ -12,57 +12,81 @@
           "input": "",
           "expected": false
         },
         {
-          "description": "pangram with only lower case",
+          "description": "handles an undefined message as empty message",
           "property": "isPangram",
-          "input": "the quick brown fox jumps over the lazy dog",
+          "input": "null",
+          "expected": false
+        },
+        {
+          "description": "recognizes a lower case pangram",
+          "property": "isPangram",
+          "input": "thequickbrownfoxjumpsoverthelazydog",
           "expected": true
         },
         {
-          "description": "missing character 'x'",
+          "description": "recognizes a missing character 'x'",
+          "property": "isPangram",
+          "input": "aquickmovementoftheenemywilljeopardizefivegunboats",
+          "expected": false
+        },
+        {
+          "description": "recognizes a missing character 'h'",
           "property": "isPangram",
-          "input": "a quick movement of the enemy will jeopardize five gunboats",
+          "input": "fiveboxingwizardsjumpquicklyatit",
           "expected": false
         },
         {
-          "description": "another missing character 'x'",
+          "description": "recognizes an upper case pangram",
           "property": "isPangram",
-          "input": "the quick brown fish jumps over the lazy dog",
+          "input": "THEQUICKBROWNFOXJUMPSOVERTHELAZYDOG",
+          "expected": true
+        },
+        {
+          "description": "recognizes a missing character in upper case",
+          "property": "isPangram",
+          "input": "THEQUICKBROWNFOXJUMPSOVERTHELAZYDOLL",
           "expected": false
         },
         {
-          "description": "pangram with underscores",
+          "description": "recognizes a mixed case pangram",
           "property": "isPangram",
-          "input": "the_quick_brown_fox_jumps_over_the_lazy_dog",
+          "input": "ThEFiVebOxInGWiZaRdSJuMpqUiCkLy",
           "expected": true
         },
         {
-          "description": "pangram with numbers",
+          "description": "recognizes a missing character in mixed case",
+          "property": "isPangram",
+          "input": "ThEFiVebOxInGWiZaRdSJuMpqUiCkL",
+          "expected": false
+        },
+        {
+          "description": "ignores other characters in correct pangram",
           "property": "isPangram",
-          "input": "the 1 quick brown fox jumps over the 2 lazy dogs",
+          "input": "Victor_jagt-zwölf.B0xkämpfer >qu3r<über;den'großen\" Sylter#Deich!",
           "expected": true
         },
         {
-          "description": "missing letters replaced by numbers",
+          "description": "ignores other characters in incorrect pangram",
           "property": "isPangram",
-          "input": "7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog",
+          "input": "Victor_jagt-zwölf.B0xkämpfer >qu3r< über;einen'großen\" Sylter#Teich!",
```

Comment:
This has the same description as the previous test.

Non ASCII test cases should not be included, see: https://github.com/exercism/problem-specifications/issues/428

Reply:
OK, I'll remove non-ASCII tests.